src/phoenix/db/models.py

```diff
@@ -93,6 +94,15 @@ class Project(Base):
     id: Mapped[int] = mapped_column(primary_key=True)
     name: Mapped[str]
     description: Mapped[Optional[str]]
+    gradient_start_color: Mapped[str] = mapped_column(
+        String,
+        server_default=func.text("'#5bdbff'"),
```

Comment:
`text` needs to come from `sqlalchemy`, not `func`
```suggestion
        server_default=text("'#5bdbff'"),
```

Reply:
Oh cool, will do